packages/slate/src/create-editor.ts

```diff
@@ -236,45 +232,45 @@ export const createEditor = (): Editor => {
         // other inline nodes, or parent blocks that only contain inlines and
         // text.
         if (isInlineOrText !== shouldHaveInlines) {
-          Transforms.removeNodes(editor, { at: path.concat(n), voids: true })
-          n--
+          Transforms.removeNodes(editor, { at: path.concat(i), voids: true })
+          break
```

Comment:
Why all the `break` statements?  The point of the loop is to continue and to use `n` to keep track of removed or added nodes so it points to the right index/node.  Was there a bug you experienced with this code as it is?